index.html

```diff
@@ -51,22 +56,203 @@ <h1 id="principal-title" class="color-blue fonts">I’m Luis Emilio. <br> Glad t
+                                <div class="frame-2">
+                                    <p class="client">CANOPY</p>
+                                    <img src="./vectors/Counter.png" alt="Counter">
+                                    <p class="Role">Back End Dev</p>
+                                    <img src="./vectors/Counter.png" alt="Counter">
+                                    <p class="Role">2023</p>
+                                </div>
+                                <p class="card-description">
+                                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                                </p>
+                                <ul class="tags-list-style-none">
+                                    <li class="tag-description1">HTML</li>
+                                    <li class="tag-description2">CSS</li>
+                                    <li class="tag-description3">JAVASCRIPT</li>
+                                </ul>
+                                <button type="button" class="see-btn">See project</button>
+                        </div>
+                    </div>
+                </div>    
+                    <div class="grid-container">
+                        <div class="card-container">
+                            <img class="card-img" src="./vectors/Snapshoot Portfolio4.png" alt="image of my project" />
+                            <div class="text-container">
+                                <h2 class="card-title">Multi-Post <br> Stories</h3>
+                                    <div class="frame-2">
+                                        <p class="client">CANOPY</p>
+                                        <img src="./vectors/Counter.png" alt="Counter">
+                                        <p class="Role">Back End Dev</p>
+                                        <img src="./vectors/Counter.png" alt="Counter">
+                                        <p class="Role">2023</p>
+                                    </div>
+                                    <p class="card-description">
+                                        A daily selection of privately personalized reads; no accounts or sign-ups
+                                        required.
+                                    </p>
+                                    <ul class="tags-list-style-none">
+                                        <li class="tag-description1">HTML</li>
+                                        <li class="tag-description2">CSS</li>
+                                        <li class="tag-description3">JAVASCRIPT</li>
+                                    </ul>
+                                    <button type="button" class="see-btn">See project</button>
+                            </div>
+                        </div>
+    </section>
+    <div class="headline">
+        <h2 id="principal-title" class="color-blue fonts">About Myself
+        </h2>
```

Comment:
- [ ] According to the project's requirements you need to use `display: flex` to lay out the content of "about me" section, please make sure to use it.

- [ ] Please implement this rounded edge (top right) 🔽 in your "about-me" section, because it's part of the design and is currently missing in your project.

![2023-03-24_232000](https://user-images.githubusercontent.com/104319462/227660477-a4d590f4-35e8-43d9-97b9-14995e3deb4d.png)

- [ ] Please note the differences between your design and the Figma design 🔽 and fix the issues to match the Figma.

## Your Design
![2023-03-24_232257](https://user-images.githubusercontent.com/104319462/227661384-9fcde959-f3e4-43e9-8183-6278308b01b5.png)

## Figma Design
![2023-03-24_232933](https://user-images.githubusercontent.com/104319462/227661378-9314b73c-bc2c-427b-8d14-497f8e4012e2.png)